blazor/dropdown-list/value-binding.md

```diff
@@ -54,7 +56,7 @@ The following code demonstrates, Value and Text field of the DropDownList compon
 
 ## Primitive type binding
 
-The DropDownList has support to load array of primitive data such as strings and numbers.
+The DropDownList has support to load array of primitive data such as strings and numbers. You can bind value of primitive data to [@bind-Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute of DropDownList 

```

Comment:
The DropDownList has support to load array of primitive data such as strings and numbers. Bind the value of primitive data to the [@bind-Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute of the DropDownList.